tests/lib/tools/tests.invariant

```diff
@@ -150,6 +150,22 @@ check_cgroup_scopes() {
 		return 0
 	fi
 
+	wait_for_system_scopes=5
+	while [ "${wait_for_system_scopes}" -gt 0 ]; do
+	    has_active_scope=0
+	    while read -r -d '' scope; do
+		if systemctl is-active "$(basename "${scope}")"; then
+		    has_active_scope=1
+		    break
+		fi
+	    done < <(find /sys/fs/cgroup/system.slice -name 'snap.*.scope' -print0)
+	    if [ "${has_active_scope}" -eq 0 ]; then
+		break
```

Comment:
```suggestion
			break
```

Reply:
Correct. I did not realize the indentation was not matching the rest of the file. I have re-indented the whole change.